learn/core_concepts/relevancy.md

```diff
@@ -97,11 +107,11 @@ The `attribute` rule sorts the results by [attribute importance](/learn/core_con
 
 For now, MeiliSearch supports two custom rules that can be added to [the ranking rules array](#behavior): one for ascending sort and one for descending sort.
 
-To add a custom ranking rule, you have to communicate either `asc` for ascending order or `desc` for descending order followed by the field name in parentheses.
+To add a custom ranking rule, you have to communicate the attribute name followed by a colon (`:`) and either `asc` for ascending order or `desc` for descending order.
 
-- To apply an **ascending sort** (results sorted by increasing value of the attribute): `asc(attribute_name)`
+- To apply an **ascending sort** (results sorted by increasing value of the attribute): `attribute_name:asc`
 
-- To apply a **descending sort** (results sorted by decreasing value of the attribute): `desc(attribute_name)`
+- To apply a **descending sort** (results sorted by decreasing value of the attribute): `attribute_name:desc`
 
 **The attribute must have a numeric value** in all of the documents contained in that index. **If any value is not a numeric type, the sorting rule won't be applied**.
```

Comment:
Not true anymore, you can apply custom ranking rules on string :)

Reply:
Great, thanks for clarifying that. I thought custom ranking rule behavior had remained the same.